server/src/servershot.js

```diff
@@ -205,6 +209,10 @@ class Shot extends AbstractShot {
         clipRewrites.rewriteShotUrls();
         const oks = clipRewrites.commands();
         const json = this.toJSON();
+        if (json.isOwner) {
+          delete json.isOwner;
```

Comment:
@6a68 I agree since we are no longer storing isOwner in database, we don't need isOwner as REGULAR_ATTR. [Here](https://github.com/mozilla-services/screenshots/pull/4966/files#diff-6db5e3f6a06e986e2aa9ab0f5c0170d1R23) it will anyways always return empty as we don't have json value in db. Will update to set it as property of shot itself, then we don't need to explicitly delete on update and insert